### DISCUSSION TITLE
Deployment question

### DISCUSSION THREAD

**Discussion by natea** [gorules/zen#105] [Q&A]
In the documentation, it describes how to deploy Zen using Helm charts on a Kubernetes cluster:
https://gorules.io/docs/cloud/deployment/kubernetes

But where can one find the Helm charts? It says on that page, "Pull the Helm charts provided by GoRules engineering team locally, copy the values.yaml to my-values.yaml file and make the modifications to it."

Does that mean that the Helm charts are not available in the public Github repo, but must be specially requested?

**Comment by stefan-gorules** [comment]
Hi @natea, thank you for the question.

The Kubernetes guidelines are for the Cloud (Self-hosted) Business rules management system by GoRules which we are working on making generally available through paid subscription. To learn more about it go to: https://gorules.io/pricing. Currently Helm charts are acquired per request.

To outline the difference between Zen engine and GoRules BRMS:
- Zen engine - open-source rules engine core that can be used to evaluate rules generated through UI (e.g. https://editor.gorules.io, https://github.com/gorules/jdm-editor - graph will be available soon, cloud, self-hosted etc.).
- GoRules BRMS - full fledge solution with user management, versioning, document-like system, audit logs, etc. that uses Zen engine under the hood to evaluate business rules.

Zen engine is basically our community open-source package. It's aim is to provide transparency, avoid vendor-locking and allow companies and individuals to build custom solutions using Zen engine (rules engine core).

**Comment by stefan-gorules** [comment]
Now available through ArtifactHub: https://artifacthub.io/packages/helm/gorules/gorules-brms.

**Comment by ivanmiletic** [comment]
Prerequisites:
* License key - You can delete the existing environment on [portal.gorules.io](http://portal.gorules.io/) and create a new one with the fresh key
* Postgresql

Artifacthub: https://artifacthub.io/packages/helm/gorules/gorules-brms

Command:

# helm install my-gorules-brms oci://[registry-1.docker.io/gorulescharts/gorules-brms](http://registry-1.docker.io/gorulescharts/gorules-brms) --version 1.2.1 -f my-values.yaml

You can find my-values.yaml on artifact hub. Edit data:
Replace appUrl and licenceKey with your variables.
Fill in the Postgres connection details
Next, you can forward the port for testing:
# kubectl port-forward services/my-gorules-brms 4200:80

BRMS will be available on http://localhost:4200/. In a real environment, you should also set ingress.

We will be updating documentation for k8s.
